<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>language_setting</class>
 <widget class="QWidget" name="language_setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="Label_Tchinese" name="label_Tchinese">
   <property name="geometry">
    <rect>
     <x>565</x>
     <y>340</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>繁體中文</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>800</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Label_return_L" name="label_return">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>490</y>
     <width>100</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_backdrop">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>220</y>
     <width>720</width>
     <height>360</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border-image: url(:/tu/ALL-IN-ONE ICON/BKG_long.png);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Label_chinese" name="label_chinese">
   <property name="geometry">
    <rect>
     <x>565</x>
     <y>260</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>简体中文</string>
   </property>
  </widget>
  <widget class="Label_save_L" name="label_save">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>490</y>
     <width>100</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="Label_english" name="label_english">
   <property name="geometry">
    <rect>
     <x>565</x>
     <y>420</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_D1">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>260</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_D2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>340</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_D3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>420</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_backdrop</zorder>
  <zorder>label_Tchinese</zorder>
  <zorder>label_return</zorder>
  <zorder>label_chinese</zorder>
  <zorder>label_save</zorder>
  <zorder>label_english</zorder>
  <zorder>label_D2</zorder>
  <zorder>label_D3</zorder>
  <zorder>label_D1</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Label_chinese</class>
   <extends>QLabel</extends>
   <header>language_setting.h</header>
  </customwidget>
  <customwidget>
   <class>Label_Tchinese</class>
   <extends>QLabel</extends>
   <header>language_setting.h</header>
  </customwidget>
  <customwidget>
   <class>Label_english</class>
   <extends>QLabel</extends>
   <header>language_setting.h</header>
  </customwidget>
  <customwidget>
   <class>Label_return_L</class>
   <extends>QLabel</extends>
   <header>language_setting.h</header>
  </customwidget>
  <customwidget>
   <class>Label_save_L</class>
   <extends>QLabel</extends>
   <header>language_setting.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
